### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by tropikoder** [just-ai/jaicf-kotlin#256] [Q&A] [answered]
Hello guys. I hope this project's devs are well.  Could someone comment on the status of this project and what are the chances it will continue going forward?

**Comment by upersuser** [comment]
One of the developers said that the framework is working stably, there are things that need to be updated, but they will not do it. The project is open to improvements from the community.

**Reply by tropikoder** [reply]
thanks!

**Comment by morfeusys** [comment, marked as answer]
@tropikoder Hey! We are on the way for the next great release with LLM support! ))
